app/forms/applicants/address_selection_form.rb

```diff
@@ -10,14 +10,19 @@ class AddressSelectionForm
 
     validates :address, presence: true
 
+    def initialize(*args)
+      attributes[:lookup_used] = true
+      super
+    end
```

Comment:
I think this will work instead of overriding `initialize`:
```
def lookup_used
  true
end
```

Reply:
OK I'll try

Reply:
Booo :sob: